webrtc.html

```diff
@@ -8777,13 +8780,27 @@ <h2 id="sec.stats-model">Statistics Model</h2>
               MUST run the following steps:</p>
               <ol>
                 <li>
-                  <p>Let <var>selectorArg</var> be the methods first
+                  <p>Let <var>selector</var> be the methods first
                   argument.</p>
                 </li>
                 <li>
-                  <p>If <var>selectorArg</var> is neither <code>null</code> nor
-                  a valid <a>selector</a>, return a promise rejected with a
-                  <code>TypeError</code>.</p>
+                  <p>Let <var>connection</var> be the
+                  <code><a>RTCPeerConnection</a></code> object on which
+                  the method was invoked.</p>
+                </li>
+                <li>
+                  <p>If <var>selector</var> is a <code>MediaStreamTrack</code>,
+                  find a sender or receiver on <var>connection</var>, which
+                  track matches <var>selector</var>, and let <var>selector</var>
+                  be that sender or receiver. If no such sender or receiver
+                  exist, throw an <code>InvalidAccessError</code> exception and
+                  abort these steps.</p>
+                </li>
+                <li>
+                  <p>If <var>selector</var> was not created by
+                  <var>connection</var>, throw an
+                  <code>InvalidAccessError</code> exception and abort these
+                  steps.</p>
```

Comment:
The problem is *selector* may be a track then. And tracks are not created by *connection*.